Update the serial port part of UP2's doc (#1938)
projectacrn/acrn-hypervisor: doc/getting-started/up2.rst

Connecting to the serial port
=============================

> The UP2 board has two serial ports and the following figure shows the bottom-side of UP2 board, 

Reviewer: ```suggestion
The UP2 board has two serial ports. The following figure shows the
UP2 board's 40-pin HAT connector we'll be using, 
```

> which is from `UP2 specifications 

Reviewer: ```suggestion
as documented in the `UP2 Datasheet 
```

<https://up-board.org/wp-content/uploads/datasheets/UP-Square-DatasheetV0.5.pdf>`_.

.. image:: images/the-bottom-side-of-UP2-board.png
   :align: center
   
We'll access the serial port through the I/O pins in the 
40-pin HAT connector using a `USB TTL serial cable
<http://www.ftdichip.com/Products/USBTTLSerial.htm>`_, 

> and introduce how to enable serial port with 

Reviewer: ```suggestion
and show how to connect a 
```
